src/sass/_Fabric.Color.Variables.scss

```diff
@@ -77,3 +77,13 @@ $ms-color-alert:             $ms-color-orange;
 $ms-color-alertBackground:   $ms-color-neutralLighter;
 $ms-color-error:             $ms-color-redDark;
 $ms-color-errorBackground:   #fde7e9;
+
+
+//== High contrast colors
+//
+$ms-color-contrastBlackDisabled: #00ff00;
```

Comment:
We'd usually document new colors, but I don't think we'd provide classes for these (e.g. `.ms-bgColor-contrastBlackDisabled`) for general use.